ci/unit_tests/functions_delete_package/main_test.py

```diff
@@ -257,3 +292,34 @@ def test_wrongNamespace(self, useApikey):
         # Fail
         self.t_exitCodeAndLogMessage(1,
         "The resource could not be found. Check your cloudfunctions url and namespace.", [paramsDelete])
+
+    @pytest.mark.parametrize('useApikey', [True])
+    def test_deletePackageByRegex(self, useApikey):
+        """Tests if functions_delete_package deletes uploaded packages by regex matching."""
+
+        params = ['-c', os.path.join(self.dataBasePath, 'exampleFunctionsEmpty.cfg'),
+                '--cloudfunctions_namespace', self.namespace,
+                '--cloudfunctions_url', self.cloudFunctionsUrl]
+
+        if useApikey:
+            params.extend(['--cloudfunctions_apikey', self.apikey])
+        else:
+            params.extend(['--cloudfunctions_username', self.username, '--cloudfunctions_password', self.password])
+
+        createdPackages = []
+        # Generate random packages with common prefix and delete them afterwards
+        for i in range(3):
+            newParams = list(params)
+            newPackage = self.packageBase + "REGEX-" + str(uuid.uuid4())
+            newParams.extend(['--cloudfunctions_package', newPackage])
+            functions_deploy.main(newParams)
+            self._checkPackageExists(newPackage)
+            createdPackages.append(newPackage)
+
+        # delete the packages
+        namePattern = '^' + self.packageBase + 'REGEX-.*'
+        params.extend(['--cloudfunctions_package_name_pattern', namePattern])
+        self.t_noException([params])
+
+        for package in createdPackages:
+            self._checkPackageDeleted(package)
```

Comment:
And you can add test, if '*' will delete all packages.

Reply:
At the end, consider adding some arguments check (as in [test_badArgs](https://github.com/IBM/watson-assistant-workbench/blob/devel/ci/unit_tests/functions_deploy/main_test.py#L226) or in [test_args_basic](https://github.com/IBM/watson-assistant-workbench/blob/devel/ci/unit_tests/workspace_delete/main_test.py#L282))

Reply:
The testing namespace contains packages from several people, not only our packages for CI. So deleting everything might not be a good idea. As for the arguments, yes, I'll add tests for that.

Reply:
Done in 235a110.